src/views/SpaceCreate.vue

```diff
@@ -235,12 +239,61 @@ watchEffect(() => {
     ];
   }
 });
+
+const stepTwoIsValid = computed(() => {
+  return (
+    dateEnd.value &&
+    dateEnd.value > dateStart.value &&
+    form.value.snapshot &&
+    form.value.snapshot > blockNumber.value / 2 &&
+    !choices.value.some((a, i) => a.text === '' && i === 0)
+  );
+});
+
+const currentStep = ref(1);
+
+const stepIsValid = computed(() => {
+  if (
+    currentStep.value === 1 &&
+    form.value.name &&
+    form.value.body.length <= bodyLimit.value &&
+    passValidation.value[0] === true
+  )
+    return true;
+  else if (currentStep.value === 2 && stepTwoIsValid.value) return true;
+  else return false;
+});
+
+const preview = ref(false);
+
+watch(preview, () => {
+  window.scrollTo(0, 0);
+});
+
+function selectDate(date) {
+  selectedDate.value = date;
+  modalDateSelectOpen.value = true;
+}
+
+// Update form start date when going to step two
+watch(currentStep, () => {
+  if (!userSelectedDateEnd.value)
+    form.value.start = parseInt((Date.now() / 1e3).toFixed());
+});
+
+import { usePlugins } from '@/composables/usePlugins';
+
+// Check if has plugins that can be confirgured on proposal creation
+const { getPluginComponents } = usePlugins();
+const createPluginComponents = computed(() =>
+  getPluginComponents('Create', Object.keys(props.space?.plugins ?? {}))
+);
```

Comment:
I think this is not needed. It's only meant to be used in `<PluginCreate>` where the components then get actually mounted. Here you just need to check `props.space?.plugins` if I'm not missing anything.

Reply:
Just checking `props.space?.plugins` we will not know if there are plugins that also need configuration on the proposal creation. I only want to show step 3 if plugins that are enabled also have configuration option of proposal creation.

Reply:
Oh yea sure, sorry. But then the `pluginIndex` from `usePlugins` should be your friend. There you can check for `defaults.proposal`.